auth/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,10 @@
             android:name="com.facebook.sdk.ApplicationId"
             android:value="@string/facebook_application_id"/>
 
+        <meta-data
```

Comment:
@samtstern cc4b64b is the best solution I can come up with. Since we are upgrading the Facebook sdk for people, they will expect the Facebook upgrade guide (remove the `initializeApp`) to work. With this latest commit, that will still work, but people will have to override our manifest if they want to disable it:
```xml
<meta-data
    android:name="com.facebook.sdk.AutoLogAppEventsEnabled"
    android:value="false"
    tools:replace="android:value"/>
```